src/vs/workbench/contrib/languageRuntime/browser/languageRuntimeActions.ts

```diff
@@ -931,6 +932,29 @@ export function registerLanguageRuntimeActions() {
 		}
 	});
 
+	registerAction2(class extends Action2 {
+		/**
+		 * Constructor.
+		 */
+		constructor() {
+			super({
+				id: LANGUAGE_RUNTIME_DISCOVER_RUNTIMES_ID,
+				title: nls.localize2('workbench.action.language.runtime.discoverAllRuntimes', "Discover All Interpreters"),
```

Comment:
We could use wording here that is more like "Refresh Interpreter Discovery", if we think that is better or more clear?